core/vm/contracts.go

```diff
@@ -121,6 +125,19 @@ var PrecompiledContractsBLS = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{18}): &bls12381MapG2{},
 }
 
+var PrecompiledContracts4788 = map[common.Address]PrecompiledContract{
```

Comment:
Why this instead of in Cancun? Now we have cancun with KZG but no beaconnroot, and we have this with beaconroot but no KZG